tests/distributed/test_new_kvstore.py

```diff
@@ -18,8 +18,8 @@
     import struct
 
 # Create an one-part Graph
-node_map = F.tensor([0, 0, 0, 0, 0, 0], F.int64)
-edge_map = F.tensor([0, 0, 0, 0, 0, 0, 0], F.int64)
+node_map = {'_N': F.tensor([[0, 6]], F.int64)}
```

Comment:
how about keep the original node/edge_map and the new one in test. then we could make sure original one will raise exception and the new one will work well.

Reply:
we need to make sure `is_range_part = False` cases could be handled: raise exception.

Reply:
Got you, but how about adding a new test case in `test_partition`? which should be more suitable.

Reply:
Got you, but keep the original node/map here will not cover the `is_range_part`, let me add a new test in test_partition.

Reply:
have you added some tests to cover the exception code path?

Reply:
see test_partition